app/views/list/_list_row_template.slim

```diff
@@ -0,0 +1,10 @@
+tr.request
+  td.request-checkbox
+    input[type="checkbox"]
```

Comment:
http://stackoverflow.com/questions/9073690/post-an-array-from-an-html-form-without-javascript

The basic scheme is that `name[][key]` adds that key to a new element in the array `name` (or the most recently created new element, if it doesn't already have a value for that key). So if you have a bunch of rows that have `requests[][selected]` and `requests[][id]` inputs, you'll get an array called `requests` with a series of elements that look like `{selected: true, id: 123}`